plugins/linux-v4l2/v4l2-output.c

```diff
@@ -95,6 +96,38 @@ bool loopback_module_available()
 	return false;
 }
 
+bool use_vcam_restart_workaround()
+{
+	static int version = -1;
+	FILE *fp;
+	size_t result = 0;
+	int v_major;
+	int v_minor;
+	int v_bugfix;
+
+	if (version >= 0)
+		return version <= 0x0D02;
+
+	fp = fopen("/sys/module/v4l2loopback/version", "r");
```

Comment:
This is not functional inside a Flatpak context, `/sys/module` is not reachable (and adding sandbox hole will not be accepted).

Reply:
Additionally, the `/version` file doesn't exist for me at all in v4l2loopback-dkms 0.12.7 on Ubuntu 24.04.

Reply:
> Additionally, the `/version` file doesn't exist for me at all in v4l2loopback-dkms 0.12.7 on Ubuntu 24.04.

Interesting, I'll double check this on a fresh 24.04 with the dkms package. In any case; if the version isn't reported in earlier loopback modules, it'll behave as it it did before.

Reply:
@WizardCM - confirmed; looks like this wasn't introduced until later version.

I'll await feedback, but I'm thinking that we need a fix that doesn't rely on detecting version. See https://github.com/obsproject/obs-studio/pull/11906#issuecomment-2727237666

Reply:
This should be fixed in 52d57cf7 - I've tested this with loopback 0.12.5, 0.12.7, 0.13.0, and 0.14.0. We don't attempt to detect the version anymore; instead we test output capabilities - if they aren't there, we try to reset them (with the STREAMON/OFF workaround), if successful, then we make sure we use the workaround each time the device is opened/closed.

Note: I believe the workaround is only needed for loopback 0.12.5-0.12.7.